doc/user/content/sql/create-source/webhook.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "CREATE SOURCE: Webhook"
+description: "Ingesting data into Materialize with HTTP requests"
+pagerank: 50
+menu:
+  main:
+    parent: 'create-source'
+    identifier: webhook
+    name: Webhook
+    weight: 40
+---
+
+{{< private-preview />}}
+
+{{% create-source/intro %}}
+Webhook sources expose a URL that allow your other applications to push data into Materialize.
+{{% /create-source/intro %}}
+
+
+## Syntax
+
+{{< diagram "create-source-webhook.svg" >}}
```

Comment:
We usually combine all these into a single graphic. Is there a reason to split them up here?

Reply:
Our other sources seemed to split things up a bit, e.g. [Kafka](https://materialize.com/docs/sql/create-source/kafka/), I can combine them if you think that's better!